packages/web/src/pages/collection-page/components/mobile/CollectionPage.tsx

```diff
@@ -159,6 +159,8 @@ const CollectionPage = ({
     metadata && metadata?.variant !== Variant.SMART
       ? metadata._is_publishing
       : false
+  const access =
```

Comment:
nit: why not just `!!metadata && 'access' in metadata ? metadata.access`?

Reply:
I personally prefer the explicit check 🤷